### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by jennresado**
- profile tiles
- redirect to contact page on click adopt
- news item is not in the details since news item is actually used as a stretch requirement for a blurb section
- images are fixed height and center, the dummy data images look off, but I implemented an image preview section on the animal page, so admin can check the picture and how it will be displayed before adding/updating animal (this add is on the animal-page branch )
- animal details are in a scroll box to prevent uneven tiles
- css
- cleaned up some navigation and footer

### CODE REVIEW

File: src/assets/Dispositions.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const dispositions = [
+    'Good with other animals', 
+    'Good with children', 
+    'Animal must be leased at all times'
```
Comment (jduell12) [inline src/assets/Dispositions.js:4]: A small typo above - Animal must be leashed at all times but instead it's being leased

File: src/components/Animal.js [hunk 2]
```diff
@@ -16,6 +20,17 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
     const [availability, setAvailability] = useState('')
     const [newsItem, setNewsItem] = useState('')
     const [description, setDescription] = useState('')
+    const objectMapping = {
+        "id": id,
+        "type": type,
+        "selectBreeds": selectBreeds,
```
Comment (jduell12) [inline src/components/Animal.js:26]: What is this selectBreeds key for?
I see that you also have a breeds key so was curious what the difference was between these two keys in this object

File: src/components/Animal.js [hunk 3]
```diff
@@ -100,27 +127,28 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
...
+                        body[key] = objectMapping[key]
+                    }
+                } else if (objectMapping[key]) {
+                    body[key] = objectMapping[key]
+                } else {
+                    body[key] = idAnimal[key]
+                }
+            }
+            
             console.log(body)
```
Comment (jduell12) [inline src/components/Animal.js:152]: Did you want to leave the console.log in for production?

File: src/components/Animal.js [hunk 4]
```diff
@@ -132,6 +160,24 @@ const Animal = ({ animalsDb, onAddAnimal, onUpdateAnimal, onDeleteAnimal }) => {
...
+                "type": type,
+                "breeds": breed,
+                "disposition": disposition,
+                "pic": picture,
+                "availability": availability,
+                "news_item": newsItem,
+                "description": description
+            }
+
             console.log(body)
```
Comment (jduell12) [inline src/components/Animal.js:181]: I think if this code it going to the production branch we probably don't need the console.logs

**Review by jduell12 — CHANGES_REQUESTED** [review]

File: src/assets/Dispositions.js (hunk 1 above)
Comment (jennresado) [reply src/assets/Dispositions.js:4]: Thanks for catching. Fixed.

File: src/components/Animal.js (hunk 2 above)
Comment (jennresado) [reply src/components/Animal.js:26]: selectBreeds is used to dynamically populate the breed choices based on selected type.

File: src/components/Animal.js (hunk 3 above)
Comment (jennresado) [reply src/components/Animal.js:152]: This is just the setup with the dummy data for now, I decided to console.log the data instead. I can remove if that's preferred.

File: src/components/Animal.js (hunk 4 above)
Comment (jennresado) [reply src/components/Animal.js:181]: Same as above.

File: src/components/Animal.js (hunk 3 above)
Comment (jennresado) [reply src/components/Animal.js]: I decided to comment it out instead.

**Review by jduell12 — APPROVED** [review]

**Pull request merged**